### PR TITLE
Add owner to KR

### PR DESCRIPTION
**Pull request by Nianwens**

### CODE REVIEW

**Comment by Nianwens** [comment]
#91

**Comment by Nianwens** [comment]
image
image

File: src/StateMangement/OKRReducers.ts [hunk 1]
```diff
@@ -49,7 +52,7 @@ export const reducer = (state: OKRMainState = initialState, action) => {
         });
         // If an area isn't already selected, set the first areas as the selected area
         draft.selectedArea = draft.selectedArea ? draft.selectedArea : draft.areas[0];
-        draft.selectedArea = draft.selectedArea ? draft.selectedArea : draft.areas[0];
+        draft.identityProvider = new IdentityProvider();
```
Comment (beccamc) [inline src/StateMangement/OKRReducers.ts]: Did you do this for any perf gain, or just for a cleaner implementation?

Comment (Nianwens) [reply src/StateMangement/OKRReducers.ts]: mainly for the cleaner implementation, we do not need to initialize the same provider 3 times.

**Pull request merged**